cdap-proto/src/main/java/io/cdap/cdap/proto/sourcecontrol/RepositoryConfig.java

```diff
@@ -144,8 +144,8 @@ public Builder setAuthType(AuthType authType) {
       return this;
     }
 
-    public Builder setTokenName(String tokenName) {
-      this.tokenName = tokenName;
+    public Builder setPasswordName(String passwordName) {
```

Comment:
this is an existing issue, but it's better not to have setters for individual AuthConfig fields and just have a setter for the entire AuthConfig. That way RepositoryConfig doesn't need to know implementation details about the unrelated AuthConfig class.

Reply:
fixing it now

Reply:
updated